lesson03/assignment/css/_settings.responsive.scss

```diff
@@ -40,12 +40,12 @@ $breakpoints: (
     "desk"          "screen and (min-width: 1024px)",
     "lg"            "screen and (min-width: 920px)",
     "tab"           "screen and (min-width: 768px)",
-    "mobile"        "screen and (max-width: 767px)",
+    "mobile"        "(max-width: 768px)", 
     "infograph"     "screen and (min-width: 540px)",
     "med"           "screen and (min-width: 40em)",
     "sm"            "screen and (min-width: 30em)",
     "small"         "screen and (min-width: 20em)",
-    "tiny"          "screen and (max-width: 320px)",
+    "phone"          "(max-width: 576px)",
```

Comment:
What does this change accomplish? Since it's an unusual choice, it makes sense to add a comment to explain why it's being used.

Reply:
For practice: I added additional comments, though looking at my own code now I am not sure why exactly I did it this way, so my assumption (in knowing myself) is that I was having trouble getting something to work and just kept banging on it till it got there.